### DISCUSSION TITLE
How to Subscribe with lambda instead of IDisposable

### DISCUSSION THREAD

**Discussion by martinrhan** [reactiveui/ReactiveUI#3647] [Q&A]
In the documentation, there are examples that when calling Subsribe function, lambdas are passed in as arguments. However, I only have IObservable<T>.Subscribe(IObserver<T>) in my IDE. What extension namespace should I include? The examples in documentation didn't provide that information.

**Comment by anaisbetts** [comment]
This is an Rx.NET question, but it's probably in either `System`,`System.Reactive`, or `System.Reactive.Linq`
